e2e-tests/cypress/integration/appeal-submission-creation-of-submission-information-file.feature

```diff
@@ -1,12 +1,12 @@
-@wip
 Feature: Appeal Submission - Creation of submission information file
 
   As a Planning Inspectorate case worker
   I want a human readable file containing the appellant’s appeal submission information, at point of submission
   So that I have all the information collected during the appeal submission that was not explicitly passed to Horizon
 
-  @as-101
+  @as-101 @as-1441
   Scenario: Summary of submission
     Given a prospective appellant has provided valid appeal information
     When the appeal is submitted
     Then a submission information file is created
+    And the cookie banner does not exist
```

Comment:
I would really liked to have tested this as a separate feature, but it isn't efficient to do so without having a known appeal fixture pre-existing for the tests.

Reply:
understood.
in my mind; "And" in a feature file translates as "a thing we strapped on here and surely we dont want to run the whole scenario again to test for it?"
suggest we review all feature files at some point and work out how to de-'and' them but i totally get why this is happening for this instance and i agree that it guards us against a thing we need guarding against..